cime_config/usermods_dirs/PLUMBER2/AR-SLu/shell_commands

```diff
@@ -0,0 +1,42 @@
+./xmlchange PLUMBER2SITE=AR-SLu
+./xmlchange PTS_LON=-66.459808
+./xmlchange PTS_LAT=-33.464802
+./xmlchange RUN_STARTDATE=2010-01-01
+./xmlchange DATM_YR_ALIGN=2010
+./xmlchange DATM_YR_START=2010
+./xmlchange DATM_YR_END=2010
+./xmlchange START_TOD=10800
+./xmlchange ATM_NCPL=48
+
+echo "CLM_USRDAT.PLUMBER2:datafiles='/glade/work/oleson/PLUMBER2/datm_files/AR-SLu/CLM1PT_data/CTSM_DATM_AR-SLu_2010-2010.nc '" >> user_nl_datm_streams 
```

Comment:
This should go into

$CSMDATA/lnd/clm2/surfdata_map/PLUMBER2

and similarly setup as NEON is now.

Where $CSMDATA is /glade/campaign/cesm/cesmdata/cseg/inputdata on Derecho

Reply:
This is #2410